databases/week2/more-queries.sql

```diff
@@ -0,0 +1,30 @@
+-- Get all tasks assigned to users whose email ends in @spotify.com
+SELECT * 
+FROM task 
+JOIN user ON task.user_id = user.id 
+WHERE user.email LIKE '%@spotify.com';
+
+-- Add table with status ENUM
+ALTER TABLE task ADD COLUMN status ENUM('Not started', 'Ongoing', 'Finished');
+
+-- Get all tasks for 'Donald Duck' with status 'Not started'
+SELECT * 
+FROM task 
+JOIN user ON task.user_id = user.id 
+WHERE user.name = 'Donald Duck' 
+AND task.status = 'Not started';
+
+-- Get all tasks for 'Maryrose Meadows' that were created in September
+SELECT * 
+FROM task 
+JOIN user ON task.user_id = user.id 
+WHERE user.name = 'Maryrose Meadows' 
+AND MONTH(task.created) = 9;
```

Comment:
Same applies for these queries: you should be using the `user_task` table, since the `task` table does not have a `user_id` field.